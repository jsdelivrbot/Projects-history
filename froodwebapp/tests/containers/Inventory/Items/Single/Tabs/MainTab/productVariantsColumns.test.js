import {
  productOptionsColumns,
  productVariantsNewColumns,
  productVariantsEditColumns
} from 'containers/Inventory/Items/Single/Tabs/MainTab/productVariantsColumns';
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

const { describe, expect, it } = global;

describe('productVariantsColumns', () => {
  const props = {
    productKeyColumnDisabled: true
  };

  it('renders the productOptionsColumns component', () => {
    const component = shallow(<productOptionsColumns { ...props } />);

    expect(component.length).toEqual(1);
  });

  it('capturing Snapshot of productOptionsColumns', () => {
    const component = shallow(<productOptionsColumns { ...props } />);
    const view = toJson(component);
    const actual = expect(view);

    actual.toMatchSnapshot();
  });

  it('renders the productVariantsNewColumns component', () => {
    const component = shallow(<productVariantsNewColumns { ...props } />);

    expect(component.length).toEqual(1);
  });

  it('capturing Snapshot of productVariantsNewColumns', () => {
    const component = shallow(<productVariantsNewColumns { ...props } />);
    const view = toJson(component);
    const actual = expect(view);

    actual.toMatchSnapshot();
  });

  it('renders the productVariantsEditColumns component', () => {
    const component = shallow(<productVariantsEditColumns { ...props } />);

    expect(component.length).toEqual(1);
  });

  it('capturing Snapshot of productVariantsEditColumns', () => {
    const component = shallow(<productVariantsEditColumns { ...props } />);
    const view = toJson(component);
    const actual = expect(view);

    actual.toMatchSnapshot();
  });
});
