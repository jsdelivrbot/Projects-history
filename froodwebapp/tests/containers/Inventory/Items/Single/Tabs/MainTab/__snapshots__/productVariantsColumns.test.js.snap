// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`productVariantsColumns capturing Snapshot of productOptionsColumns 1`] = `
<productOptionsColumns
  productKeyColumnDisabled={true}
/>
`;

exports[`productVariantsColumns capturing Snapshot of productVariantsEditColumns 1`] = `
<productVariantsEditColumns
  productKeyColumnDisabled={true}
/>
`;

exports[`productVariantsColumns capturing Snapshot of productVariantsNewColumns 1`] = `
<productVariantsNewColumns
  productKeyColumnDisabled={true}
/>
`;
